tcf_website/static/course/javascript/course_data.js

```diff
@@ -1,9 +1,14 @@
 const loadData = data => {
+    /* eslint-disable camelcase */
     const { a_plus, a, a_minus, b_plus, b, b_minus, c_plus, c, c_minus, d_plus, d, d_minus, f, withdraw, drop } = data;
     const grades_data = [a_plus, a, a_minus, b_plus, b, b_minus, c_plus, c, c_minus, d_plus, d, d_minus, f, withdraw, drop];
+    /* eslint-enable camelcase */
 
     createChart(grades_data);
 
+    // these ternary-hack if statements are not favored by eslint
```

Comment:
This was the way it was when this file was first created:
https://github.com/thecourseforum/theCourseForum2/commit/6ff038c7546aa87cf3e6b5b6add3d24486c36a9b#diff-8a9ab61568853b41f8cea34b0750299ce453b73b16a02a2b068d0a116c815427
But I was going to refactor this file, too. Here's the link to my PR: https://github.com/thecourseforum/theCourseForum2/pull/341/files#diff-8a9ab61568853b41f8cea34b0750299ce453b73b16a02a2b068d0a116c815427R15-R64.
FYI I'm the only one following this PR (because I commented on it), so you might want to post this on `#eng-chat` or `#code-reviews` for others' opinions.